### ISSUE TITLE
ember-template-lint throws warnings in latest beta release

### ISSUE THREAD

**Issue by stopfstedt** [ilios/frontend#4120]
we want that upgrade, tho.

fix all the warnings.

**Comment by stopfstedt** [comment]
```

```$ yarn lint:hbs
yarn run v1.10.1
$ ember-template-lint app/templates
/home/stefan/dev/projects/frontend/app/templates/components/assign-students.hbs
  20:12  error  Incorrect indentation of element modifier 'action' beginning at L20:C12. Expected 'action' to be at L28:C6.  attribute-indentation
  28:4  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L28:C4. Expected '<button>' to be at L29:C4.  attribute-indentation
  63:14  error  Incorrect indentation of element modifier 'action' beginning at L63:C14. Expected 'action' to be at L66:C12.  attribute-indentation
  66:10  error  Incorrect indentation of close bracket '>' for the element '<td>' beginning at L66:C10. Expected '<td>' to be at L67:C10.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/curriculum-inventory-report-list.hbs
  41:12  error  Incorrect indentation of element modifier 'action' beginning at L41:C12. Expected 'action' to be at L45:C10.  attribute-indentation
  45:8  error  Incorrect indentation of close bracket '>' for the element '<td>' beginning at L45:C8. Expected '<td>' to be at L46:C8.  attribute-indentation
  48:12  error  Incorrect indentation of element modifier 'action' beginning at L48:C12. Expected 'action' to be at L52:C10.  attribute-indentation
  52:8  error  Incorrect indentation of close bracket '>' for the element '<td>' beginning at L52:C8. Expected '<td>' to be at L53:C8.  attribute-indentation
  55:12  error  Incorrect indentation of element modifier 'action' beginning at L55:C12. Expected 'action' to be at L59:C10.  attribute-indentation
  59:8  error  Incorrect indentation of close bracket '>' for the element '<td>' beginning at L59:C8. Expected '<td>' to be at L60:C8.  attribute-indentation
  62:12  error  Incorrect indentation of element modifier 'action' beginning at L62:C12. Expected 'action' to be at L66:C10.  attribute-indentation
  66:8  error  Incorrect indentation of close bracket '>' for the element '<td>' beginning at L66:C8. Expected '<td>' to be at L67:C8.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/dashboard-myreports.hbs
  7:16  error  Incorrect indentation of element modifier 'action' beginning at L7:C16. Expected 'action' to be at L9:C10.  attribute-indentation
  9:8  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L9:C8. Expected '<button>' to be at L10:C8.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/detail-stewards.hbs
  7:9  error  Incorrect indentation of element modifier 'action' beginning at L7:C9. Expected 'action' to be at L9:C6.  attribute-indentation
  9:4  error  Incorrect indentation of close bracket '>' for the element '<div>' beginning at L9:C4. Expected '<div>' to be at L10:C4.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/ilios-users.hbs
  22:16  error  Incorrect indentation of element modifier 'action' beginning at L22:C16. Expected 'action' to be at L23:C10.  attribute-indentation
  22:115  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L22:C115. Expected '<button>' to be at L24:C8.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/learnergroup-bulk-finalize-users.hbs
  43:12  error  Incorrect indentation of element modifier 'action' beginning at L43:C12. Expected 'action' to be at L46:C6.  attribute-indentation
  46:4  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L46:C4. Expected '<button>' to be at L47:C4.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/learnergroup-cohort-user-manager.hbs
  86:10  error  Incorrect indentation of element modifier 'action' beginning at L86:C10. Expected 'action' to be at L88:C4.  attribute-indentation
  88:2  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L88:C2. Expected '<button>' to be at L89:C2.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/learnergroup-upload-data.hbs
  106:10  error  Incorrect indentation of element modifier 'action' beginning at L106:C10. Expected 'action' to be at L109:C4.  attribute-indentation
  109:2  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L109:C2. Expected '<button>' to be at L110:C2.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/learnergroup-user-manager.hbs
  173:14  error  Incorrect indentation of element modifier 'action' beginning at L173:C14. Expected 'action' to be at L175:C8.  attribute-indentation
  175:6  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L175:C6. Expected '<button>' to be at L176:C6.  attribute-indentation
  182:14  error  Incorrect indentation of element modifier 'action' beginning at L182:C14. Expected 'action' to be at L184:C8.  attribute-indentation
  184:6  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L184:C6. Expected '<button>' to be at L185:C6.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/locale-chooser.hbs
  2:10  error  Incorrect indentation of element modifier 'action' beginning at L2:C10. Expected 'action' to be at L9:C4.  attribute-indentation
  9:2  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L9:C2. Expected '<button>' to be at L10:C2.  attribute-indentation
  15:16  error  Incorrect indentation of element modifier 'action' beginning at L15:C16. Expected 'action' to be at L22:C10.  attribute-indentation
  22:8  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L22:C8. Expected '<button>' to be at L23:C8.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/my-profile.hbs
  82:18  error  Incorrect indentation of element modifier 'action' beginning at L82:C18. Expected 'action' to be at L85:C12.  attribute-indentation
  85:10  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L85:C10. Expected '<button>' to be at L86:C10.  attribute-indentation
  100:18  error  Incorrect indentation of element modifier 'action' beginning at L100:C18. Expected 'action' to be at L103:C12.  attribute-indentation
  103:10  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L103:C10. Expected '<button>' to be at L104:C10.  attribute-indentation
  110:18  error  Incorrect indentation of element modifier 'action' beginning at L110:C18. Expected 'action' to be at L112:C12.  attribute-indentation
  112:10  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L112:C10. Expected '<button>' to be at L113:C10.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/new-directory-user.hbs
  125:14  error  Incorrect indentation of element modifier 'action' beginning at L125:C14. Expected 'action' to be at L132:C8.  attribute-indentation
  132:6  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L132:C6. Expected '<button>' to be at L133:C6.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/new-user.hbs
  171:12  error  Incorrect indentation of element modifier 'action' beginning at L171:C12. Expected 'action' to be at L178:C6.  attribute-indentation
  178:4  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L178:C4. Expected '<button>' to be at L179:C4.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/program-list.hbs
  18:12  error  Incorrect indentation of element modifier 'action' beginning at L18:C12. Expected 'action' to be at L21:C10.  attribute-indentation
  21:8  error  Incorrect indentation of close bracket '>' for the element '<td>' beginning at L21:C8. Expected '<td>' to be at L22:C8.  attribute-indentation
  24:12  error  Incorrect indentation of element modifier 'action' beginning at L24:C12. Expected 'action' to be at L27:C10.  attribute-indentation
  27:8  error  Incorrect indentation of close bracket '>' for the element '<td>' beginning at L27:C8. Expected '<td>' to be at L28:C8.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/programyear-competencies.hbs
  9:9  error  Incorrect indentation of element modifier 'action' beginning at L9:C9. Expected 'action' to be at L11:C6.  attribute-indentation
  11:4  error  Incorrect indentation of close bracket '>' for the element '<div>' beginning at L11:C4. Expected '<div>' to be at L12:C4.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/school-competencies-expanded.hbs
  5:9  error  Incorrect indentation of element modifier 'action' beginning at L5:C9. Expected 'action' to be at L7:C6.  attribute-indentation
  7:4  error  Incorrect indentation of close bracket '>' for the element '<div>' beginning at L7:C4. Expected '<div>' to be at L8:C4.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/school-leadership-expanded.hbs
  2:7  error  Incorrect indentation of element modifier 'action' beginning at L2:C7. Expected 'action' to be at L4:C4.  attribute-indentation
  4:2  error  Incorrect indentation of close bracket '>' for the element '<div>' beginning at L4:C2. Expected '<div>' to be at L5:C2.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/school-session-attributes-manager.hbs
  13:17  error  Incorrect indentation of element modifier 'action' beginning at L13:C17. Expected 'action' to be at L16:C12.  attribute-indentation
  16:10  error  Incorrect indentation of close bracket '>' for the element '<input>' beginning at L16:C10. Expected '<input>' to be at L17:C10.  attribute-indentation
  36:17  error  Incorrect indentation of element modifier 'action' beginning at L36:C17. Expected 'action' to be at L39:C12.  attribute-indentation
  39:10  error  Incorrect indentation of close bracket '>' for the element '<input>' beginning at L39:C10. Expected '<input>' to be at L40:C10.  attribute-indentation
  49:17  error  Incorrect indentation of element modifier 'action' beginning at L49:C17. Expected 'action' to be at L52:C12.  attribute-indentation
  52:10  error  Incorrect indentation of close bracket '>' for the element '<input>' beginning at L52:C10. Expected '<input>' to be at L53:C10.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/school-session-type-form.hbs
  124:14  error  Incorrect indentation of element modifier 'action' beginning at L124:C14. Expected 'action' to be at L127:C8.  attribute-indentation
  127:6  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L127:C6. Expected '<button>' to be at L128:C6.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/school-session-types-expanded.hbs
  2:7  error  Incorrect indentation of element modifier 'action' beginning at L2:C7. Expected 'action' to be at L4:C4.  attribute-indentation
  4:2  error  Incorrect indentation of close bracket '>' for the element '<div>' beginning at L4:C2. Expected '<div>' to be at L5:C2.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/school-vocabularies-expanded.hbs
  1:5  error  Incorrect indentation of element modifier 'action' beginning at L1:C5. Expected 'action' to be at L3:C2.  attribute-indentation
  3:0  error  Incorrect indentation of close bracket '>' for the element '<div>' beginning at L3:C0. Expected '<div>' to be at L4:C0.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/school-vocabularies-list.hbs
  75:41  error  Incorrect indentation of element modifier 'action' beginning at L75:C41. Expected 'action' to be at L76:C18.  attribute-indentation
  76:16  error  Incorrect indentation of close bracket '>' for the element '<span>' beginning at L76:C16. Expected '<span>' to be at L77:C16.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/user-menu.hbs
  1:8  error  Incorrect indentation of element modifier 'action' beginning at L1:C8. Expected 'action' to be at L7:C2.  attribute-indentation
  7:0  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L7:C0. Expected '<button>' to be at L8:C0.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/user-profile-bio.hbs
  4:21  error  Incorrect indentation of element modifier 'action' beginning at L4:C21. Expected 'action' to be at L5:C6.  attribute-indentation
  5:4  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L5:C4. Expected '<button>' to be at L6:C4.  attribute-indentation
  13:24  error  Incorrect indentation of element modifier 'action' beginning at L13:C24. Expected 'action' to be at L14:C6.  attribute-indentation
  14:4  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L14:C4. Expected '<button>' to be at L15:C4.  attribute-indentation
  106:56  error  Incorrect indentation of element modifier 'action' beginning at L106:C56. Expected 'action' to be at L107:C10.  attribute-indentation
  107:8  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L107:C8. Expected '<button>' to be at L108:C8.  attribute-indentation

/home/stefan/dev/projects/frontend/app/templates/components/user-profile-ics.hbs
  7:26  error  Incorrect indentation of element modifier 'action' beginning at L7:C26. Expected 'action' to be at L8:C6.  attribute-indentation
  8:4  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L8:C4. Expected '<button>' to be at L9:C4.  attribute-indentation
  16:24  error  Incorrect indentation of element modifier 'action' beginning at L16:C24. Expected 'action' to be at L17:C6.  attribute-indentation
  17:4  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L17:C4. Expected '<button>' to be at L18:C4.  attribute-indentation
  22:21  error  Incorrect indentation of element modifier 'action' beginning at L22:C21. Expected 'action' to be at L24:C6.  attribute-indentation
  24:4  error  Incorrect indentation of close bracket '>' for the element '<button>' beginning at L24:C4. Expected '<button>' to be at L25:C4.  attribute-indentation

✖ 80 problems (80 errors, 0 warnings)

**Comment by stopfstedt** [comment]
closed via #4119
